embedded-oracle-xe/src/test/java/com/playtika/test/oracle/EmbeddedOracleBootstrapConfigurationTest.java

```diff
@@ -72,13 +74,19 @@ void shouldSaveAndGetUnicode() {
         assertThat(jdbcTemplate.queryForObject("select name from employee where id = 1", String.class)).isEqualTo("some data \uD83D\uDE22");
     }
 
+    @org.junit.Test
```

Comment:
Be careful about using `@org.junit.Test` it won't run on jupiter engine. Use ` org.junit.jupiter.api.Test` here